distro.py

```diff
@@ -511,6 +511,17 @@ def distro_release_attr(attribute):
     return _distro.distro_release_attr(attribute)
 
 
+class cached_property(object):
```

Comment:
I think we should add docstrings here to explain what we're doing.

Reply:
done!